src/common/sdpa_types.hpp

```diff
@@ -33,6 +34,20 @@ namespace impl {
 #define DNNL_ARG_VALUES DNNL_ARG_SRC_2
 #define DNNL_ARG_ATTN_MASK DNNL_ARG_SHIFT
 
+/// Types of attention mask
+typedef enum {
+    dnnl_attn_mask_buffer,
```

Comment:
I'd use undef value as a guard. And please assign fixed values to them.
Minor documentation on each of them wouldn't hurt.